corehq/apps/reports/templates/reports/form/edit_submission.html

```diff
@@ -2,9 +2,11 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
+{% block js-inline %}{{ block.super }}
+{% include 'cloudcare/includes/touchforms-inline.html' %}
+{% endblock %}
```

Comment:
this should be fine. before the `js` block was auto compressing everything in it which was causing the issue in the file you linked to. the `js-inline` clock does not get compressed so it should work though is an unfortunate necessary hack